### PR TITLE
Fix #631

### LINKED ISSUE #631: _msize is not provided as an override?
Cross-posting from [Z3](https://github.com/Z3Prover/z3/issues/6890) and grepping through the source-code doesn't seem to provide an override for it

### PR DESCRIPTION
**Pull request by Naville**
Add wrapper override for the Windows variant of maloc_usable_size : _msize

### CODE REVIEW

File: src/snmalloc/override/malloc.cc [hunk 1]
```diff
@@ -40,6 +40,12 @@ extern "C"
   {
     return snmalloc::libc::malloc_usable_size(ptr);
   }
+  SNMALLOC_EXPORT 
+  size_t SNMALLOC_NAME_MANGLE(_msize)( 
+    MALLOC_USABLE_SIZE_QUALIFIER void* ptr) 
+  { 
+    return snmalloc::libc::malloc_usable_size(ptr); 
+  }
```
Comment (mjp41) [inline src/snmalloc/override/malloc.cc:48]: Thanks for doing this.  If you could put it in a `ifdef` that would be great.  Also clang-format is complaining about the whitespace changes. The below suggestions should fix both.

```suggestion

#ifdef _WIN32
  SNMALLOC_EXPORT
  size_t SNMALLOC_NAME_MANGLE(_msize)(
    MALLOC_USABLE_SIZE_QUALIFIER void* ptr)
  {
    return snmalloc::libc::malloc_usable_size(ptr);
  }
#endif
```

**Review by mjp41 — CHANGES_REQUESTED** [review]
Thanks for doing this.  Sorry I have been swamped by another project so didn't have time to replicate your issue.

**Comment by Naville** [comment]
Done

File: src/snmalloc/override/malloc.cc [hunk 2]
```diff
@@ -41,6 +41,15 @@ extern "C"
     return snmalloc::libc::malloc_usable_size(ptr);
   }
 
+#ifdef _WIN32
+  SNMALLOC_EXPORT
+  size_t SNMALLOC_NAME_MANGLE(_msize)(
+    MALLOC_USABLE_SIZE_QUALIFIER void* ptr)
```
Comment (mjp41) [inline src/snmalloc/override/malloc.cc:47]:
```suggestion
  size_t SNMALLOC_NAME_MANGLE(_msize)(MALLOC_USABLE_SIZE_QUALIFIER void* ptr)
```

**Review by mjp41 — APPROVED** [review]

**Pull request merged**